source/tests/system/nidcpower_session_tests.cpp

```diff
@@ -156,7 +158,7 @@ TEST_F(NiDCPowerSessionTest, InvalidSession_CloseSession_ReturnsInvalidSessionEr
   EXPECT_TRUE(status.ok());
   EXPECT_EQ(kInvalidDCPowerSession, response.status());
   std::string error_message = get_error_message(response.status());
-  EXPECT_STREQ(kInvalidDCPowerSessionMessage, error_message.c_str());
+  EXPECT_THAT(error_message.c_str(), HasSubstr(kInvalidDCPowerSessionMessage));
```

Comment:
Curious: Was this failing on Windows, Ubuntu, or both?

Reply:
On Ubuntu, IIRC.